### PR TITLE
Test Update for GA Release changes. 

### PR DESCRIPTION
**Pull request by RudraSharma93Microsoft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/support/azext_support/custom.py [hunk 1]
```diff
@@ -49,7 +52,7 @@ def update_support_tickets(cmd, client,
     body["country"] = contact_country
     body["preferred_support_language"] = contact_language
 
-    return client.update(support_ticket_name=ticket_name, severity=severity, contact_details=body)
+    return client.update(support_ticket_name=ticket_name, update_support_ticket=body)
```
Comment (mit2nil) [inline src/support/azext_support/custom.py]: This doesn't look correct. Did updated SDK changed the interface?

Comment (RudraSharma93Microsoft) [reply src/support/azext_support/custom.py]: Yes it did, this is how the update function takes input now

Comment (RudraSharma93Microsoft) [reply src/support/azext_support/custom.py]: Created the SDK 2 mins back

File: src/support/azext_support/tests/latest/recordings/test_support_tickets.yaml [hunk 2]
```diff
@@ -556,34 +542,41 @@ interactions:
...
+        ticket from python cli test. Do not assign and close after a day.","serviceId":"/providers/Microsoft.Support/services/517f2da6-78fd-0498-4e22-ad26996b1dfc","serviceDisplayName":"Billing","status":"Open","problemStartTime":"2020-03-09T15:09:46Z","createdDate":"2020-03-09T22:09:51Z","modifiedDate":"2020-03-09T22:09:53Z"}},{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Support/supportTickets/120030521002318","name":"120030521002318","type":"Microsoft.Support/supportTickets","properties":{"description":"Request
+        Summary / New Limit: \nResource Manager, WESTUS2, Av2 Series / 101","problemClassificationId":"/providers/Microsoft.Support/services/06bfd9d3-516b-d5c6-5802-169c800dec89/problemClassifications/e12e3d1d-7fa0-af33-c6d0-3c50df9658a3","problemClassificationDisplayName":"Compute-VM
+        (cores-vCPUs) subscription limit increases","severity":"Moderate","require24X7Response":false,"enrollmentId":"","contactDetails":{"firstName":"Ayush","lastName":"Goyal","preferredContactMethod":"Email","primaryEmailAddress":"aygoya@microsoft.com","preferredTimeZone":"Pacific
```
Comment (mit2nil) [inline src/support/azext_support/tests/latest/recordings/test_support_tickets.yaml]: Use some other test account where we do not have any pii from the team to generate the yaml.

**Comment by mit2nil** [comment]
I do not see update to the validator file. Can you address resource id validator bug as well?

**Pull request merged**